### PR TITLE
Update GitHub Pages index view to be Documentation Landing Page MVP

### PR DESCRIPTION
**Pull request by billfienberg**
### Screenshot

doc-landing-page-mvp

### Testing Instructions

1. Navigate to an HTML previewer (for example: https://htmlpreview.github.io/)
1. Paste the link to the HTML file into the HTML previewer
   - HTML link: https://github.com/department-of-veterans-affairs/va.gov-team/blob/doc-landing-page-v1/docs/index.html
1. Click **Preview**
1. Check out the landing page

### Ticket

https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/6299

### Description

This PR updates the current `va.gov-team` GitHub Pages index view from an `index.md` to an `index.html`. The content in the `index.html` represents the MVP of our documentation landing page. 

Since this is an MVP, we wanted to focus on speed and function-over-form. I felt the fastest way to validate our idea would be to build a static HTML file. I attempted to use our Design System, but that resulted in me spending time worrying about dependencies/building/deploying, and where all of that code should live. 

I asked Megan for some direction, and she said it wasn't necessary to use the Design System for the MVP. 

Ultimately, I chose to use a utility-first CSS framework called [Tailwinds](https://tailwindcss.com/) so I could focus 100% on building the page. 

The following paragraph from [Tailwind's docs](https://tailwindcss.com/docs/utility-first/) succinctly explains why I went with a utility-first CSS framework:

> But once you’ve actually built something this way, you’ll quickly notice some really important benefits:
> 
> - **You aren’t wasting energy inventing class names.** No more adding silly class names like sidebar-inner-wrapper just to be able to style something, and no more agonizing over the perfect abstract name for something that’s really just a flex container.
> 
> - **Your CSS stops growing.** Using a traditional approach, your CSS files get bigger every time you add a new feature. With utilities, everything is reusable so you rarely need to write new CSS.
> 
> - **Making changes feels safer.** CSS is global and you never know what you’re breaking when you make a change. Classes in your HTML are local, so you can change them without worrying about something else breaking. - https://tailwindcss.com/docs/utility-first/

### Tiny URL (old content from `index.md` file until this PR gets merged)

https://tinyurl.com/vagovdocs

### Tradeoffs/Gotchas

- I asked Megan if we should incorporate Andrew's original content from the `index.md` file, and she said not to worry about it. 
- According to Jon W. on the Analytics team, [the link type `class`-es should be at start of `class` string](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/6354#issuecomment-601422757).
- To move as quickly as possible, I chose to use Tailwinds via a CDN. Unfortunately, [using Tailwinds via CDN](https://tailwindcss.com/docs/installation#using-tailwind-via-cdn) comes with the following tradeoffs:
  - You can't customize [Tailwind's default theme](https://tailwindcss.com/docs/customizing-colors#default-color-palette)
  - You can't use any [directives](https://tailwindcss.com/docs/functions-and-directives/) like `@apply`, `@variants`, etc.
  - You can't enable features like [`group-hover`](https://tailwindcss.com/docs/pseudo-class-variants/#group-hover)
  - You can't install third-party plugins

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,737 @@
...
+            you work.
+          </div>
+        </div>
+        <div class="beta-banner bg-blue-300 px-40 py-4">
+          <span class="font-bold text-xl">
+            This is a beta site.
+          </span>
+          We are always looking to make improvements.
+          <span class="font-bold">
+            Send us your feedback.
```
Comment (rianfowler) [inline docs/index.html]: what's this going to do?

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,737 @@
...
+        j.async = true;
+        j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, "script", "dataLayer", "GTM-T2ZTDXZ");
+    </script>
+    <!-- End Google Tag Manager -->
+    <title>VSP Documentation Landing Page</title>
+    <meta charset="UTF-8" />
+    <link
+      href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css"
```
Comment (rianfowler) [inline docs/index.html]: one day we'll have a drop in css for the va design system

**Review by rianfowler — APPROVED** [review]
LGTM- i think future state we should make it responsive but that's a low priority. 
Screen Shot 2020-03-20 at 13 29 42

File: docs/index.html (hunk 1 above)
Comment (billfienberg) [reply docs/index.html]: Content & IA is still figuring out what they want that link to do. So it's just a placeholder for now.

File: docs/index.html (hunk 2 above)
Comment (billfienberg) [reply docs/index.html]: That makes sense.

**Comment by billfienberg** [comment]
> LGTM- i think future state we should make it responsive but that's a low priority.
> Screen Shot 2020-03-20 at 13 29 42

Agreed. I asked about responsiveness at the beginning, and was told it wasn't necessary for MVP.

**Pull request merged**